inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Please put  "#!/bin/bash" after the licenses.

Reply:
fixed